src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import getPics from './redux/api';
+import Navbar from './components/Navbar';
+import PicsPage from './pages/PicsPage';
+import './App.css';
+import SinglePic from './pages/SinglePic';
+
+function App() {
+  const [selected] = useState('all');
+  const dispatch = useDispatch();
+
+  /*
+  const handleClick = (event) => {
+    console.log(event.target.value);
+    // navigate('/rockets/');
+  }
+
+handleClick={handleClick}
+*/
```

Comment:
- [ ] [OPTIONAL] Please clean your comment on your code base!

Reply:
Change done. Thank you @mukhammadrizooff